extensions/s3/src/test/java/io/deephaven/extensions/s3/testlib/SingletonContainers.java

```diff
@@ -25,6 +25,11 @@ public final class SingletonContainers {
     // https://java.testcontainers.org/test_framework_integration/manual_lifecycle_control/#singleton-containers
     // https://testcontainers.com/guides/testcontainers-container-lifecycle/#_using_singleton_containers
 
+    public static String S3_ENDPOINT = "s3.endpoint";
+    public static String CLIENT_REGION = "client.region";
+    public static String S3_ACCESS_KEY_ID = "s3.access-key-id";
+    public static String S3_SECRET_ACCESS_KEY = "s3.secret-access-key";
```

Comment:
As mentioned earlier, we should delete these too, expose static methods to get the values directly.